NEWS.md

```diff
@@ -1,6 +1,8 @@
 # Rgemini `0.5.1`
 * Improved query efficiency in `mlaps` and other functions for clinical derived variables
 * Removed duplicated code in `icu_entry()`
+* New function `homelessness_flag` which creates a table indicating homelessness status of genc_ids
```

Comment:
Oh sorry, one additional comment: Can you add this to a new section above (since this function will only be released with the next version of Rgemini, which will probably be `0.5.2` but you can just put `develop` for now until we've finalized the new version update).

```suggestion
# Rgemini `develop`
* New function `homelessness_flag` which creates a table indicating homelessness status of genc_ids

# Rgemini `0.5.1`
* Improved query efficiency in `mlaps` and other functions for clinical derived variables
* Removed duplicated code in `icu_entry()`
```